releng/org.eclipse.n4js.targetplatform/org.eclipse.n4js.targetplatform.target

```diff
@@ -75,7 +74,7 @@
       <unit id="com.google.inject" version="3.0.0.v201605172100"/>
       <unit id="io.github.classgraph" version="4.8.35.v20190528-1517"/>
       <unit id="javax.xml.bind" version="2.2.0.v201105210648"/>
-      <unit id="javax.xml.stream" version="1.0.1.v201004272200"/>
+      <unit id="javax.xml.stream" version="1.0.1.v201004272200"/> <!-- Oomph want's to remove javax.xml.stream, but it's required by rest-assured-2.4.0.jar in tester.tests.helper (tested July 2019) -->
```

Comment:
try again with the extraUnits Annotation in the targlet-generator and use "," separation and only one entry.....